source/docs/advanced/payments/magento2-headless-payments.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
I would have put this file in the Magento2 section instead of the Payments section. Not sure if that's the correct move.

Reply:
:shrug: Why not… we'll refactor if needed!